helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -662,7 +683,7 @@ fluentd:
     # Prefix for pos_file tail source parameter
     # Can be used if you want to run multiple instances of fluentd on the same host
     # https://docs.fluentd.org/input/tail#pos_file-highly-recommended
-    posFilePrefix: /var/log/splunk-fluentd
+    posFilePrefix: /var/addon/splunk/fluent_pos/splunk-fluentd
```

Comment:
Is `/var/run` a more conventional place for uses like this?

Reply:
Otel checkpoint is already stored in `/var/addon/splunk/` directory so the same convention was used here.